braindecode/datasets/base.py

```diff
@@ -57,16 +57,15 @@ class WindowsDataset(BaseDataset):
     description: pandas.Series
         holds additional info about the windows
     """
+    md_keys = ['i_supercrop_in_trial', 'i_start_in_trial', 'i_stop_in_trial']
```

Comment:
This seems a bit confusing, why defined at that place? Why not as before? Also md_keys seems confusing as this is keys to get indices, but tehre are appartenly other metadatas as well, like `target`

Reply:
I've changed this with the newest commit so that the list is initialized in the init instead. Although this is probably not an issue, having it there means the get method doesn't have to create the same list every time it's called. Also it could eventually let us add an argument to let the user specify the columns to return instead of using the default ones.